WalletWasabi/Blockchain/TransactionBuilding/ChangelessTransactionCoinSelector.cs

```diff
@@ -96,12 +90,6 @@ internal static bool TryGetCoins(SelectionStrategy strategy, long target, Dictio
 
 		if (solution is not null)
 		{
-			// Sanity check: do not return solution that is much higher or much lower than the target.
-			if (solution.Sum() > target * MaxExtraPayment || solution.Sum() < target * MinPaymentThreshold)
-			{
-				return false;
-			}
```

Comment:
This did not take into account lesser strategy.

Reply:
What does this mean?

Reply:
Nevermind. I got it. It was completely moved.